testsuite/integration-arquillian/servers/eap7/pom.xml

```diff
@@ -46,16 +28,8 @@
         <profile>
             <id>auth-server-eap7</id>
             <properties>
-                <keycloak.server.home>${project.build.directory}/unpacked/jboss-iam-1.0</keycloak.server.home>
+                <keycloak.server.home>${project.build.directory}/unpacked/RH-SSO-7.0</keycloak.server.home>
```

Comment:
This should not be 'RH-SSO-7.0' it's still Keycloak, just deployed onto EAP 7. There shouldn't be references to RH-SSO in community.